sycl/test/acap/test_aie_global.cpp

```diff
@@ -0,0 +1,82 @@
+// REQUIRES: acap
+
+// RUN: %acap_clang %s -o %s.bin
+// RUN: %add_acap_result %s.bin
+// RUN: rm %s.bin
+
+#define TRISYCL_DEVICE_ALLOCATOR_DEBUG
+
+/// Example of using neighbor memory tiles
+#include <sycl/sycl.hpp>
+
+#include "triSYCL/vendor/Xilinx/graphics.hpp"
+#include "triSYCL/vendor/Xilinx/acap/aie/service.hpp"
+#include "triSYCL/vendor/Xilinx/acap/aie/log.hpp"
+
+#include "triSYCL/vendor/Xilinx/acap/aie/device_libstdcpp.hpp"
+#include <cstring>
+#include <iostream>
+#include <vector>
+
+using namespace sycl::vendor::xilinx;
+using namespace sycl::vendor::xilinx;
+using namespace trisycl::vendor::xilinx;
+
+struct A {
+  static constexpr unsigned size = 5;
+  const char* str;
+  int arr[size];
+  A(const char* s) {
+    str = s;
+    acap::multi_log(str, "\n");
+  }
+  void access() {}
+  ~A() {
+    acap::multi_log(str, "\n");
+  }
+};
+
+A a{"a"};
+A b{"b"};
+A c{"c"};
+A d{"d"};
+A e{"e"};
+A f{"f"};
```

Comment:
What happen if these are also used by the host? How does it works from the device point-of-view? Where are they allocated?

Reply:
Have you looked that having such a global variable semantics on the device does not lead to some memory bloatware on the titles just because the DCE did not trigger for some reasons (think about some cycle between global variables in `std::` library or whatever)?